### ISSUE TITLE
Input Tensors need gradients?

### ISSUE THREAD

**Issue by ildoonet** [clovaai/CutMix-PyTorch#2]
https://github.com/clovaai/CutMix-PyTorch/blob/master/train.py#L240

Does input tensors need gradients?

**Comment by hellbell** [comment]
@ildoonet 
I don't think it is necessary. 
Please PR after testing without 'requires_grad' term. 
Thanks.
